examples/platforms/simulation/virtual_time/platform-sim.c

```diff
@@ -312,4 +312,25 @@ void otSysProcessDrivers(otInstance *aInstance)
 #endif
 }
 
+#if OPENTHREAD_CONFIG_OTNS_ENABLE
+
+#include <openthread/platform/alarm-micro.h>
```

Comment:
Looks like we don't actually need this header but include it to force the linker using the `otPlatOtnsStatus` defined here, right?

As a work-around, I think we should leave a comment (`TODO` or `FIXME`) here to make the intention clear.

Reply:
I copied it from the original `virtual_time/otns.c` to here. There is no intention to use it to force linker to use `otPlatOtnsStatus` defined here. I think I can remove it.

Reply:
removed.